Cargo.toml

```diff
@@ -42,15 +42,15 @@ all-types = [
     "with-bit-vec-0_6",
```

Comment:
Could you please update these features according to https://github.com/sfackler/rust-postgres/blob/master/tokio-postgres/Cargo.toml#L31?